addons/website_blog/controllers/main.py

```diff
@@ -167,6 +171,80 @@ def tags_list(tag_ids, current_tag):
         response = request.website.render("website_blog.blog_post_short", values)
         return response
 
+    @http.route(['/blog/<model("blog.blog"):blog>/<any("atom","rss"):feed_type>'], type='http', auth="public", website=True)
```

Comment:
@rim-odoo  yes you are right, not any type of render templates then i remove it <code>website = true</code>